front/app/components/CustomFieldsForm/InputIQ/index.tsx

```diff
@@ -17,8 +24,13 @@ const InputIQ = ({ phase }: { phase: IPhaseData | undefined }) => {
     phase?.attributes.similarity_enabled && isInputIQEnabled
   );
 
-  const titleMultiloc = watch('title_multiloc');
-  const bodyMultiloc = watch('body_multiloc');
+  let titleMultiloc, bodyMultiloc;
```

Comment:
It's not a good practice to re-assign variables like this inside the component. Also fine to watch both at the same time in this case, I think, and I should not create issues.